Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md

```diff
@@ -4,60 +4,68 @@ versionFrom: 8.0.0
 :::
 
 ![Homepage Template with Retrospect HTML](images/figure-14-homepage-template-with-Retrospect-html-v8.png)
 
-Click the **_Save_** button.
+Click the [Save] button.
 
-We now have a template. That's two out of the three stages complete for our first page.
+We now have a **_template_**. That's two out of the three stages complete for our first page.
 
 # **Creating Your First Content Node**
 
-Our third and final stage to creating our first page in Umbraco is to create a content node along with the Document Type and Template, to serve up an HTML page to web visitors.
+Our third and final stage to creating our first page in Umbraco, is to create a **_content node_** that uses our **_document type_** and **_template_**, to serve up an HTML page to web visitors.
 
-We're now ready to create our first page in Umbraco!   Click the **_Content_** button (first option in the top menu).
+We're now ready to create our first page in Umbraco!   Click the [Content] button (first option in the top menu).
 
-Hover over the grey text **_CONTENT_**  and you'll see the three dots **_..._** - click this.  If you've done everything correctly so far you should see the option to create our Homepage!
+Hover over the grey text [CONTENT]  and you'll see the three dots [...] - click this.  If you've done everything correctly so far you should see the option to create our Homepage!
 
 ![Create a Homepage](images/figure-15-create-a-homepage-v8.png)
 
 :::tip
-If you can't see this then don't panic - check that **_Settings> Document Types > Homepage_**  > **_Structure tab_** > **_Allow at root_** is checked.
+If you can't see this then don't panic - check that [Settings] > [Document Types] > [Homepage]  > [Structure tab] > [Allow at root] is checked.
 :::
 
-Let's create our homepage. Click the icon and you'll see what we've been setting up - our document type is now going to drive our homepage content – it gives us and the editors the fields they need.
+Click the "HomePage" **_node_** and you'll see what we've been setting up - our **_document type_** is now going to drive our homepage content – it gives us and the editors the fields they need.
 
-In the field at the top of the page you'll see "_Enter a name..._" click this and enter the name for your new content node.  We're going to call this "_Homepage_".
+In the field at the top of the page you'll see "_Enter a name..._" click this and enter the name for your new content node.  We're going to call this "Homepage".
+:::tip
+This Name will show up in the node list and will be used to create a url for the page. Try to keep it short but descriptive.
+:::
 
 ![Create a Homepage](images/figure-15a-create-a-homepage-enter-name-v8.png)
 
-Fill in the following on the **_Contents_** tab:
+Fill in the following **_properties_** underneath "Contents":
+
+```Page Title```
```

Comment:
I think we should keep these on the same line. That makes it clearer exactly what you need to change, right? 😁

Reply:
I thought so to, but then you have a problem when a property value needs one than more line, like in an RTE value. I tried multiple layouts and this seemed the most concise. I am open to suggestions tough, or if you decide it should always be on one line, sure.

Reply:
Ah, that's a very good point!! 🤔 Didn't think about that..
Let's keep it the way you made it 😁